src/tennisSimulator/Menu.java

```diff
@@ -0,0 +1,68 @@
+package tennisSimulator;
+
+import java.util.Scanner;
+
+public class Menu {
+
+    public static PlayersDatabase playersDatabase = new PlayersDatabase();
+    public static Scanner scanner = new Scanner(System.in);
+
+    public void runTennisSimulator() {
```

Comment:
Don't really like the naming since this method clearly does not run tennis simulator. It more seems like it opens the menu or something like this.